bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
running `yarn jest` requires the user to add `"jest": "jest" in its scripts. It's better to call `node_modules/.bin/jest` I think